servers.json

```diff
@@ -1370,5 +1370,12 @@
         "addresses": [
             "dropthem.eu"
         ]
+    },
+    {
+        "id": "ymzc",
+        "name": "YMZC Network",
+        "addresses": [
+            "mc.ymzc.pro"
```

Comment:
Change it from `mc.ymzc.pro` to `ymzc.pro`

Reply:
ok

Reply:
> Change it from `mc.ymzc.pro` to `ymzc.pro`

changed <3